Packages/MIES/MIES_Configuration.ipf

```diff
@@ -1788,11 +1789,9 @@ static Function CONF_RestoreHeadstageAssociation(panelTitle, jsonID, midExp)
 			PGC_SetAndActivateControl(panelTitle, "popup_Settings_Pressure_dev", str = NONE)
 		elseif(type == JSON_OBJECT)
 			ampSerial = JSON_GetVariable(jsonID, jsonHSPath + EXPCONFIG_JSON_AMPSERIAL)
-			if(IsFinite(ampSerial))
-				ampIndex = WhichListItem(num2istr(ampSerial), ampSerialList)
-				ASSERT(ampIDTracker[ampIndex] <= 2, "More than two headstages are configured for the same amplifier serial.")
-				PGC_SetAndActivateControl(panelTitle, "popup_Settings_Amplifier", val = CONF_FindAmpInList(ampSerial, ampIDTracker[ampIndex]))
-				ampIDTracker[ampIndex] += 1
+			ampChannel = JSON_GetVariable(jsonID, jsonHSPath + EXPCONFIG_JSON_AMPCHANNEL)
+			if(!IsFinite(ampSerial) || !IsFinite(ampChannel))
```

Comment:
I think this check is wrong. Both need to be finite or?

Reply:
Yes, I fixed that.